packages/blocks/src/page-block/edgeless/components/toolbar/connector/connector-tool-button.ts

```diff
@@ -91,7 +93,7 @@ export class EdgelessConnectorToolButton extends WithDisposable(LitElement) {
 
   override render() {
     const type = this.edgelessTool?.type;
-
+    const arrowColor = type === 'connector' ? 'currentColor' : '#77757D';
```

Comment:
Does the design team require to set it as #77757D in both dark/light mode? Maybe it is better to use var(--affine-icon-color) here instead of #77757D?

Reply:
Yes, it's by design.